Common/Data/Format/IniFile.h

```diff
@@ -8,13 +8,46 @@
 #include <memory>
 #include <map>
 #include <string>
+#include <string_view>
 #include <vector>
 #include <cstdint>
 
 #include "Common/File/Path.h"
 
 class VFSInterface;
 
+class ParsedIniLine {
+public:
+	ParsedIniLine() {}
+	ParsedIniLine(std::string_view key, std::string_view value) {
+		this->key = key;
+		this->value = value;
+	}
+	ParsedIniLine(std::string_view key, std::string_view value, std::string_view comment) {
+		this->key = key;
+		this->value = value;
```

Comment:
Is it intentional that this has no `this->comment = comment;`?

Though, it seems like following the current convention of `comment_` would be better than a new convention here since they're private anyway.

-[Unknown]

Reply:
First I didn't include a constructor with a comment, this is just a typo.